### PR TITLE
feat(seamless limits): add seamless/crossborder limits to all desired flows

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
This PR brings all new limits work

- [x] Use new seamless endpoint for Send flow
- [x] Sell flow

Other flows will be addressed in follow up PRs

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/Flyout/Banners/UpgradeToGold.tsx [hunk 1]
```diff
@@ -88,6 +89,23 @@ const UpgradeToGoldBanner = ({ limits, verifyIdentity }: Props) => {
     hideBanner((prevValue) => !prevValue)
   }
 
+  // find effective limit
+  let effectiveLimit = {} as LimitWithEffective
+  if (current?.daily?.effective) {
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/components/Flyout/Banners/UpgradeToGold.tsx:94]: This is a good use-case for a switch case.

Comment (blockdylanb) [reply packages/blockchain-wallet-v4-frontend/src/components/Flyout/Banners/UpgradeToGold.tsx:94]: and/or a function that can go in a model file somewhere.

**Review by macrs — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/components/Flyout/Banners/UpgradeToGold.tsx (hunk 1 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/components/Flyout/Banners/UpgradeToGold.tsx:94]: yeah this is basically dirty working solution, will improve it

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts [hunk 2]
```diff
@@ -1364,6 +1365,10 @@ type MessagesType = {
   'modals.sendBch.firststep.networkfee': 'Network Fee'
   'modals.sendBch.firststep.share_tooltip': 'Add a note to remind yourself what this transaction relates to. This note will be private and only seen by you.'
   'modals.sendBch.firststep.towallet': 'To'
+  'modals.send.banner.title': 'Uprade to Gold. Send More Crypto.'
+  'modals.send.banner.description': 'Verify your ID now and unlock Gold level trading. Send up to {dayCurrencySymbol}{dayAmount} a day.'
+  'modals.send.banner.get_started': 'Get Stared'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts:1370]: typo. `Get Started`

**Review by schnogz — CHANGES_REQUESTED** [review]

File: packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts (hunk 2 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts]: fixed 👍

**Review by schnogz — APPROVED** [review]

**Pull request merged**